doc/whats_new/v0.21.rst

```diff
@@ -179,6 +179,10 @@ Support for Python 3.4 and below has been officially dropped.
   prediction step, thus capping the memory usage. :issue:`13283` by
   `Nicolas Goix`_.
 
+- |Enhancement| :class:`ensemble.IsolationForest` now exposes ``warm_start``
+  parameter, allowing iterative addition of trees to an isolation 
+  forest. :issue:`13451` by :user:`Peter Marko <petibear>`.
```

Comment:
Should be PR number 13496

Reply:
corrected (and remembered that one puts PR# here), thx